src/test/scala/uk/gov/hmrc/test/ui/pages/section6/WarehousePage.scala

```diff
@@ -44,15 +46,13 @@ object WarehousePage extends BasePage {
   //   No  => fillPage(no)
   //   Yes => fillPage(yes, "R1234567GB")
 
-  override def fillPage(values: String*): Unit = {
+  override def fillPage(values: String*): Unit =
     if (!isClearance) {
       fillTextBoxById("identificationNumber", values(identifierOnNonClearance))
       store(WarehouseHouse -> Detail(values(identifierOnNonClearance)))
-    }
-    else if (selectYesOrNoRadio(values(yesNo))) {
+    } else if (isClearance) {
+      selectYesOrNoRadio(values(yesNo))
       fillTextBoxById("identificationNumber", values(identifierOnClearance))
       store(WarehouseHouse -> Detail(values(identifierOnClearance)))
-    }
-    else store(WarehouseHouse -> Detail(Constants.no))
-  }
+    } else store(WarehouseHouse -> Detail(Constants.no))
```

Comment:
Something wrong in the logic used in this method. Please reconsider (or we can have a chat if you need)

Reply:
ya i  changed the logic, please review